include/deal.II/lac/arpack_solver.h

```diff
@@ -139,13 +157,20 @@ class ArpackSolver : public Subscriptor
 
   /**
    * Solve the generalized eigensprectrum problem $A x=\lambda B x$ by calling
-   * the <code>dneupd</code> and <code>dnaupd</code> functions of ARPACK.
+   * the <code>dsaupd</code> and <code>dseupd</code> or
+   * <code>dnaupd</code> and <code>dneupd</code> functions of ARPACK.
    *
    * The function returns a vector of eigenvalues of length <i>n</i> and a
-   * vector of eigenvectors, where the latter should be twice the size of the
-   * eigenvalue vector. The first <i>n</i> vectors in
-   * <code>eigenvectors</code> will be the real parts of the eigenvectors, the
-   * second <i>n</i> the imaginary parts.
+   * vector of eigenvectors of length <i>n</i> in the symmetric case
+   * and of length <i>n+1<i> in the non-symmetric case. In the symmetric case
+   * all eigenvectors are real. In the non-symmetric case complex eigenvalues
+   * always occur as complex conjugate pairs. Therefore the eigenvector for a
+   * eigenvalue with nonzero complex part is stored by putting the real and
+   * the imaginary parts in consecutive columns. The eigenvector of the complex
+   * conjugate eigenvalue does not need to be stored, since it is just the complex
+   * conjugate of the stored eigenvector. If the last eigenvalue has a nonzero
```

Comment:
i don't understand the last sentence. One can't have this info prior the solution results, right? Unless you mean something else than `n+1` discussion above, i would remove it to avoid confusion.